Week-1/cattyboard/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CattyBoard Top 100 Single Charts(11.06.36)</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.11.2/css/all.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <nav>
+        <div class="fleximgs">
+            <img src="https://img.icons8.com/ios-filled/50/000000/menu-rounded.png" />
+            <img src="https://img.icons8.com/ios-filled/50/000000/home.png" />
+            <img src="https://img.icons8.com/ios-filled/50/000000/search--v1.png" />
+            <img src="https://img.icons8.com/ios-filled/50/000000/speaker.png" />
+            <img src="https://img.icons8.com/ios-filled/50/000000/user.png" />
+            <img src="https://img.icons8.com/ios-filled/50/000000/settings.png" />
+            <img src="https://img.icons8.com/ios-filled/50/000000/spotify.png" />
+            <img src="https://img.icons8.com/ios-filled/50/000000/soundcloud.png" />
+        </div>
+    </nav>
+    <header class="inline">
+
+        <img id="albumimg" src="2Eq--cattyboard.jpg" alt="album image">
+
+        <div class="container">
+
+            <div>
+                <h2>CattyBoard Top 100 Single Charts (11.06.36)</h2>
+                <p>
+                    Unknown Artist <br>
+                    2016. Charts. 100 Songs
+                </p>
+            </div>
+
+            <div class="innerflex">
+
+                <p>
+                    <span> &#9654; Play all </span>
+
+                    <span> &#43; Add to </span>
+
+                    <span> &#8230; More </span>
+                </p>
+            </div>
+        </div>
+    </header>
+    <section class="inline">
+        <table>
```

Comment:
thead and tbody ?

Reply:
There is no need for "thead". I've added "tbody".